webui/react/src/components/Navigation.tsx

```diff
@@ -35,7 +35,7 @@ const NavigationItem: React.FC<ItemProps> = ({ path, status, ...props }: ItemPro
   useEffect(() => setIsActive(location.pathname === path), [ location.pathname, path ]);
 
   const link = (
-    <Link className={classes.join(' ')} path={path} {...props}>
+    <Link className={classes.join(' ')} disabled={isActive} path={path} {...props}>
```

Comment:
@hkang1 I've added this like discussed yesterday. Seems to be working fine. What do you think?

Reply:
**nit:** can you also modify `Naviagition.module.scss` to update:
```
  .navItem.active {
    background-color: var(--theme-colors-base-0);
    cursor: default;
  }
```
to disable the pointer cursor when it is "active", that way the user doesn't think it's a broken link.

Reply:
Yes, good idea :)